catalyst_voices_packages/catalyst_cardano/catalyst_cardano/test/wallet-automation/Earthfile

```diff
@@ -1,5 +1,5 @@
 VERSION 0.8
-IMPORT github.com/input-output-hk/catalyst-ci/earthly/flutter:v3.1.25 AS flutter-ci
+IMPORT github.com/input-output-hk/catalyst-ci/earthly/flutter:v3.2.04 AS flutter-ci
```

Comment:
curious, why 04 and not just 4?

Reply:
We just followed the conversation which we had before this release.